src/systems/sensors/Sensors.cc

```diff
@@ -549,6 +702,54 @@ void Sensors::Configure(const Entity &/*_id*/,
   if (_sdf->HasElement("ambient_light"))
     this->dataPtr->ambientLight = _sdf->Get<math::Color>("ambient_light");
 
+  // Get the global illumination technique and its parameters, if specified
+  if (_sdf->HasElement("global_illumination"))
+  {
+    if (engineName != "ogre2")
+    {
+      gzerr << "Global illumination is only supported by the ogre2 render engine" << std::endl;
+    }
+    else
+    {
+      auto giElem = _sdf->FindElement("global_illumination");
+      std::string giType = giElem->GetAttribute("type")->GetAsString();
+      if (giType == "vct")
+      {
+        this->dataPtr->giVctParameters.enabled = giElem->Get<bool>("enabled", this->dataPtr->giVctParameters.enabled).first;
```

Comment:
wrap lines to 80char

Reply:
Checked style with codecheck in f82c68e8b6a8e75ea5b01d2e168c7666f12b46f6